src/components/common/TmSeed.vue

```diff
@@ -1,5 +1,11 @@
 <template>
   <div class="seed-phrase">
+    <div class="seed-warning">
+      <p>
+        This backup code is all that is needed to access your account. Please
```

Comment:
No mention of word order anymore?

Reply:
maybe this can be improved. but i think the numbers are quite clear. and most will copy to clipboard because of the button.